Packs/ANYRUN/Integrations/ANYRUN/ANYRUN_test.py

```diff
@@ -128,3 +134,30 @@ def test_taskid_from_url(self):
         url = 'https://www.madeup.com/madeup/tasks/'  # disable-secrets-detection
         url += 'this-is-the-task-id/blah/&someotherstuff'
         assert taskid_from_url(url) == 'this-is-the-task-id'
+
+
+def http_mock(method, url, verify=USE_SSL, headers=None):
+    if not headers:
+        raise ValueError("Permission denied")
+    else:
+        the_response = requests.Response()
+        the_response.status_code = 200
+        the_response._content = b'{ "image" : "image" }'
+        return the_response
+
+
+def test_get_image(mocker):
+    """
+    Given:
+        A mock response from the API
+    When:
+        Calling images_from_report
+    Then:
+        Validate that the http request gets called with authorization (headers)
+    """
+    response = {'data': {'analysis': {'content': {'screenshots': [{'permanentUrl': 'www.test.com'}]}}}}
+    mocker.patch('requests.request', side_effect=http_mock)
+    try:
+        images_from_report(response)
+    except ValueError as e:
```

Comment:
what is the test here? you want to validate that you have or don't have headers?
if you don't expect the function to raise an error, remove the except.

If you want the function to raise an error, use `with pytest.raises`. See `docs/tutorial-integration/YodaSpeak/Integrations/YodaSpeak/YodaSpeak_test.py:64`